docs/source/contributing.rst

```diff
@@ -0,0 +1,54 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+Development install
+-------------------
+
+**Note:** You will need :ref:`Node.js <https://nodejs.org/>` and :ref:`Docker <https://www.docker.com/>` to build the extension package.
```

Comment:
Unless I missed something, I believe we don't need Docker to build jupytergis locally

```suggestion
**Note:** You will need :ref:`Node.js <https://nodejs.org/>` to build the extension package.
```